src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -27,261 +27,281 @@
+          {% if requirements.failOptional is defined %}
+            {% if requirements.failOptional == false %}
+              <p>
+                <strong>{{ 'Optional parameters:'|trans }}</strong>
+                <span class="text-success">{{ 'OK'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+              </p>
+            {% else %}
+              <p>
+                <strong>{{ 'Optional parameters:'|trans }}</strong>
+                <span class="text-danger">{{ 'Please fix the following error(s)'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+              </p>
+              <ul>
+                {% for key, value in requirements.testsOptional %}
+                  {% if 'fail' == value %}
+                    <li>{{ requirements.testsErrors[key] }}</li>
+                  {% endif %}
+                {% endfor %}
+              </ul>
+            {% endif %}
+          {% endif %}
         </div>
+      </div>
     </div>
+  </div>
 </div>
 {% if system.notHostMode %}
-    <div class="card">
-        <h3 class="card-header">
-            <i class="material-icons">info_outline</i> {{ 'List of changed files'|trans }}
-        </h3>
-        <div class="card-block">
-            <div class="card-text" id="changedFiles">
-                <i class="material-icons">loop</i> {{ 'Checking files...'|trans({}, 'Admin.Advparameters.Notification') }}
-            </div>
-        </div>
+  <div class="card">
+    <h3 class="card-header">
+      <i class="material-icons">info_outline</i> {{ 'List of changed files'|trans }}
+    </h3>
+    <div class="card-block">
+      <div class="card-text" id="changedFiles">
+        <i class="material-icons">loop</i> {{ 'Checking files...'|trans({}, 'Admin.Advparameters.Notification') }}
+      </div>
     </div>
+  </div>
 {% endif %}
 {% if system.notHostMode %}
-    <script>
-        $(document).ready(function()
-        {
-            var translations = {
-                missing: '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
-                updated: '{{ "Updated files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
-                changesDetected: '{{ "Changed/missing files have been detected."|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
-                noChangeDetected: '{{ "No change has been detected in your files."|trans({}, "Admin.Advparameters.Notification")|e('js') }}'
-            };
+  <script>
+    $(document).ready(function()
+    {
+      var translations = {
+        missing: '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+        updated: '{{ "Updated files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+        changesDetected: '{{ "Changed/missing files have been detected."|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+        noChangeDetected: '{{ "No change has been detected in your files."|trans({}, "Admin.Advparameters.Notification")|e('js') }}'
```

Comment:
Watch the commas (`"`), they look different from the usual `'`.

Reply:
I fixed the quotes in 7e609646b007eba4e2e1a87926c3305f85cbd744